addons/mail/static/src/core/common/attachment_list.xml

```diff
@@ -9,90 +9,88 @@
-                            <i class="fa fa-trash"/>
-                        </div>
-                        <div t-if="canDownload(attachment) and !ui.isSmall" class="btn btn-sm btn-dark rounded opacity-75 opacity-100-hover mt-auto" t-on-click.stop="() => this.onClickDownload(attachment)" title="Download">
-                            <i class="fa fa-download"/>
-                        </div>
-                    </div>
-                </div>
-            </div>
-            <div class="d-flex flex-grow-1 flex-wrap mt-1 mx-1" t-att-class="{
+        >   
+            <div class="d-flex flex-grow-1 flex-wrap mx-1 align-items-center" t-att-class="{
                 'justify-content-end': isInChatWindowAndIsAlignedRight and !env.inComposer,
             }">
-                <!-- t-attf-class overridden in extensions -->
-                <div t-foreach="nonImagesAttachments" t-as="attachment" t-key="attachment.id"
-                    class="o-mail-AttachmentCard d-flex rounded mb-1 me-1 mw-100 overflow-auto"
-                    t-att-class="{
-                        'ms-1': isInChatWindowAndIsAlignedRight,
-                        'me-1': !isInChatWindowAndIsAlignedRight,
-                        'o-viewable': attachment.isViewable,
-                        'o-isUploading': attachment.uploading,
-                    }"
-                    t-attf-class="bg-300"
-                    t-att-title="attachment.name ? attachment.name : undefined" role="menu" t-att-aria-label="attachment.name"
-                    t-on-click="() => this.fileViewer.open(attachment, props.attachments)"
-                >
-                    <!-- o_image from mimetype.scss -->
-                    <t t-ref="nonImageMain">
-                        <div class="o-mail-AttachmentCard-image o_image flex-shrink-0 m-1" role="menuitem" aria-label="Preview" t-att-tabindex="-1" t-att-aria-disabled="false" t-att-data-mimetype="attachment.mimetype"/>
-                        <div class="overflow-auto d-flex justify-content-center flex-column px-1">
-                            <div t-if="attachment.name" class="text-truncate" t-out="props.messageSearch?.highlight(attachment.name) ?? attachment.name"/>
-                            <small t-if="attachment.extension" class="text-uppercase" t-esc="attachment.extension"/>
-                        </div>
-                    </t>
-                    <div class="o-mail-AttachmentCard-aside position-relative rounded-end overflow-hidden" t-att-class="{ 'o-hasMultipleActions d-flex flex-column': attachment.isDeletable and !env.inComposer }">
-                        <div t-if="attachment.uploading" class="d-flex justify-content-center align-items-center w-100 h-100" title="Uploading">
+                <t t-foreach="this.props.attachments" t-as="attachment" t-key="attachment.id">
+                    <div t-if="attachment.isImage" t-att-aria-label="attachment.filename"
```

Comment:
(minor) for tags with single line attribute, we should have all of them on a new line, not make a mix